lib/dialects/abstract/query-generator.js

```diff
@@ -423,6 +423,11 @@ const QueryGenerator = {
   },
 
   nameIndexes(indexes, rawTablename) {
+    if (typeof rawTablename === 'object') {
```

Comment:
You should be able to just call quoteTable https://github.com/gabegorelick/sequelize/blob/693bad925cf57302bda56d5d387942ea7f54fc20/lib/dialects/abstract/query-generator.js#L660

That will include the schema if defined - I guess for multi-tenancy with schemas you might want the name of the 
schema in the index name

Reply:
@janmeier I think you'd have to update `quoteTable` so it didn't use `.` as a separator. That makes sense when quoting table names for `FROM` clauses, but not for index names.

Reply:
Right, never mind then :-)